actions/transactions/record.go

```diff
@@ -61,3 +61,19 @@ func (a *Action) record(c *gin.Context) {
 	contract := mappings.MapToTransactionContract(transaction)
 	c.JSON(http.StatusCreated, contract)
 }
+
+// recordTransaction will save and complete a transaction
+// Record transaction godoc
+// @Summary		Record transaction
+// @Description	Record transaction
+// @Tags		New Transactions
+// @Produce		json
+// @Param		RecordTransaction body RecordTransaction true "Transaction to be recorded"
+// @Success		201 {object} models.Transaction "Created transaction"
+// @Failure		400	"Bad request - Error while parsing RecordTransaction from request body or xpub not found"
+// @Failure 	500	"Internal Server Error - Error while recording transaction"
+// @Router		/v1/transactions [post]
+// @Security	x-auth-xpub
+func (a *Action) recordTransaction(c *gin.Context) {
```

Comment:
As this is only an alias for the method above, and the only difference is the path, I would recommend using single method with swaggo feature to declare several paths.
You can do this by replacing line:
```go
// @Router		/v1/transaction/record [post]
```
with lines:
```go
// @Router		/v1/transactions [post]
// @DeprecatedRouter		/v1/transaction/record [post]
```

Reply:
I prefer to have two methods. @DeprecatedRouter doesn't allow for additional comments.